### ISSUE TITLE
PyPi package includes stale files

### ISSUE THREAD

**Issue by emontnemery** [jstasiak/python-zeroconf#250]
zeroconf PyPi package seems to include stale `zeroconf.py` from version 0.23.0, example:

```(venv) erik@DESKTOP-A7438J1:/mnt/d/development/github/home-assistant_fork$ pip3 install --upgrade --force-reinstall zeroconf==0.25.1
Collecting zeroconf==0.25.1
  Using cached https://files.pythonhosted.org/packages/ea/e7/6b006639302b1a0dc9ee9b2e9366da5c93a2eaf7a94c180755c033cf96e1/zeroconf-0.25.1-py3-none-any.whl
Collecting ifaddr (from zeroconf==0.25.1)
Installing collected packages: ifaddr, zeroconf
  Found existing installation: ifaddr 0.1.6
    Uninstalling ifaddr-0.1.6:
      Successfully uninstalled ifaddr-0.1.6
Successfully installed ifaddr-0.1.6 zeroconf-0.25.1
(venv) erik@DESKTOP-A7438J1:/mnt/d/development/github/home-assistant_fork$ pip show -f zeroconf
Name: zeroconf
Version: 0.25.1
Summary: Pure Python Multicast DNS Service Discovery Library (Bonjour/Avahi compatible)
Home-page: https://github.com/jstasiak/python-zeroconf
Author: Paul Scott-Murphy, William McBrine, Jakub Stasiak
Author-email: UNKNOWN
License: LGPL
Location: /mnt/d/development/github/home-assistant_fork/venv/lib/python3.7/site-packages
Requires: ifaddr
Files:
  __pycache__/zeroconf.cpython-37.pyc
  zeroconf-0.25.1.dist-info/COPYING
  zeroconf-0.25.1.dist-info/INSTALLER
  zeroconf-0.25.1.dist-info/METADATA
  zeroconf-0.25.1.dist-info/RECORD
  zeroconf-0.25.1.dist-info/WHEEL
  zeroconf-0.25.1.dist-info/top_level.txt
  zeroconf.py  <--- This is version 0.23.0
  zeroconf/__init__.py
  zeroconf/__pycache__/__init__.cpython-37.pyc
  zeroconf/__pycache__/test.cpython-37.pyc
  zeroconf/py.typed
  zeroconf/test.py
```

**Comment by jstasiak** [comment]
I can confirm this is an issue caused by a stray `build/lib/zeroconf.py` file in my copy of the project from which I release it – I just dropped it and the next releases won't have it. I don't think it breaks anything? Python seems to prefer the package `zeroconf/__init__.py` to the simple module.

**Comment by jstasiak** [comment]
Also: this only affects wheels.

**Comment by emontnemery** [comment]
Thanks!

I agree Python seems to prefer the package zeroconf/__init__.py to the simple module so this should not have caused anything to break.

Do you want to keep this open until next release?

**Comment by jstasiak** [comment]
Yeah, I think let's keep it open for now.

**Comment by jstasiak** [comment]
I just released 0.26.2 so let me close this.
